src/createServer.js

```diff
@@ -1,3 +1,68 @@
+/* eslint-disable max-len */
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase.js');
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function isValidCase(someCase) {
+  return cases.includes(someCase);
+}
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const Case = url.searchParams.get('toCase');
```

Comment:
Don't start variable names with capital letters, they are reserved for class names, etc.
```suggestion
    const сase = url.searchParams.get('toCase');
```